File: python-test/src/sms/testing_py/integration-test-1.py
from playsms.gateway import GatewayPlugin

class InfobipGatewayPlugin(GatewayPlugin):

    def __init__(self, config):
        super().__init__(config)

        self.username = config["infobip_username"]
        self.password = config["infobip_password"]
        self.api_key = config["infobip_api_key"]

    def send_sms(self, from_, to, text):
        client = infobip.ApiClient(
            username=self.username,
            password=self.password,
            api_key=self.api_key,
        )

        response = client.send_sms_message(
            from_=from_,
            to=to,
            text=text,
        )

        if response.status_code == 200:
            return True
        else:
            return False

File: php-test/src/sms/testing_php/integration_test-1.php
<?php

require_once "infobip/ApiClient.php";

class InfobipGatewayPlugin extends GatewayPlugin {

    public function __construct(Config $config) {
        parent::__construct($config);

        $this->username = $config["infobip_username"];
        $this->password = $config["infobip_password"];
        $this->api_key = $config["infobip_api_key"];
    }

    public function sendSms(string $from, string $to, string $text) {
        $client = new ApiClient(
            $this->username,
            $this->password,
            $this->api_key,
        );

        $response = $client->sendSmsMessage(
            $from,
            $to,
            $text,
        );

        if ($response->getStatusCode() == 200) {
            return true;
        } else {
            return false;
        }
    }

}